test/surf-extraction.Tests.ps1

```diff
@@ -188,7 +188,7 @@ document1.pdf                       `$5.50
         Invoke-Extractor $filePattern $extractorName -Format "csv"
 
         Get-Content $document2OutputFile | ConvertFrom-ExtractionCsv `
-          | where { $_.file -like "*document2.pdf" -and $_.amount -eq "`$5.50" } `
+          | where { $_.file -like "*document2.pdf" -and $_.amount -eq "`$5.50|`$4" } `
```

Comment:
❓ Not $4.00? (Ditto below.)

Reply:
no, it's not formatted by the extractor, purely the text present in the pdf.